compose/ui/ui/src/uikitMain/kotlin/androidx/compose/ui/scene/UIKitComposeSceneLayersHolder.uikit.kt

```diff
@@ -40,11 +40,13 @@ internal class UIKitComposeSceneLayersHolder(
     useSeparateRenderThreadWhenPossible: Boolean
 ) {
     val hasInvalidations: Boolean
-        get() = layers.any { it.hasInvalidations }
+        get() = _layers.any { it.hasInvalidations }
+
+    private val _layers = mutableListOf<UIKitComposeSceneLayer>()
+    val layers: List<UIKitComposeSceneLayer> get() = _layers
```

Comment:
Exposing this doesn't seem right. Why do you need it?

Reply:
It breaks another part. See `layersCache` - it should be used for enumeration everywhere to avoid exceptions during changing.

I beleive it shouldn't be exposed directly. Please consider to expose more high-level function to check required condition

Reply:
Okay, Done!